src/IdentityManager/WebApi/Controllers/AccountController.cs

```diff
@@ -0,0 +1,186 @@
+﻿using IdentityManager.API.Identity;
+using IdentityManager.API.Models;
+using IdentityManager.WebApi.Models.Response;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace IdentityManager.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AccountController : ControllerBase
+    {
+        private readonly IAuthenticationService _authenticationService;
+        private readonly IConfiguration _configuration;
+
+        public AccountController(IAuthenticationService authenticationService, IConfiguration configuration)
+        {
+            _authenticationService = authenticationService;
+            _configuration = configuration;
+        }
+
+        /// <summary>
+        /// RegisterAsync
+        /// </summary>
+        /// <param name="model"></param>
+        /// <returns></returns>
+        [HttpPost("Register")]
+        public async Task<IActionResult> RegisterAsync([FromBody] RegisterViewModel model)
+        {
+            if (ModelState.IsValid)
+            {
+                var result = await _authenticationService.RegisterUserAsync(model);
+                if (result.IsSuccess)
+                    return Ok(result);
+
+                return BadRequest(result);
+            }
+
+            return BadRequest("Some properties are not valid");
+        }
+        /// <summary>
+        /// LoginAsync
+        /// </summary>
+        /// <param name="model"></param>
+        /// <returns></returns>
+        [HttpPost("Login")]
+        public async Task<IActionResult> LoginAsync([FromBody] LoginViewModel model)
```

Comment:
We don't need this action, since Id4 server has its own Authentication endpoing.